packages/react-server/src/ReactFlightServer.js

```diff
@@ -479,12 +483,20 @@ export function resolveModelToJSON(
         request.pendingChunks++;
         const moduleId = request.nextChunkId++;
         emitModuleChunk(request, moduleId, moduleMetaData);
-        return serializeIDRef(moduleId);
+        if (parent[0] === REACT_ELEMENT_TYPE && key === '1') {
```

Comment:
What does `1` mean here? Is this because it's a tuple?

Reply:
yea